doc/content/reference/api/authentication.md

```diff
@@ -22,6 +22,8 @@ API keys are the simplest way of authorization. API keys do not expire, are revo
 - Gateway API keys
 - Organization API keys
 
+{{< warning >}} When authorizing an API call, make sure to use the API key **value** and not the API key **ID**. Otherwise, you might face an error resembling `error:pkg/auth:token (invalid token)`. {{</ warning >}}
```

Comment:
I don't think this is clear enough, users may not know the difference between **value** and **ID**. Maybe we should explain the format of our tokens first:

```
┌───┬─ Token Type (NNSXS: API key; MFRWG: OAuth access token)
NNSXS.U4H3ZFFCMSR42BUAZPW2UWGFBV4WCNI5EXDJXDY.SHIF3PP5PBMJNZESN5XLR5TZJTJUIGKVUTM2I22IVBUVCD6VIQIA
      └─ Token ID ──────────────────────────┘ └─ Token Secret ───────────────────────────────────┘
```

and that they need to include the **entire thing** when authenticating.

and that they need to censor that last part in screenshots and such.